addons/survey/static/src/js/survey.js

```diff
@@ -129,6 +129,15 @@ if(!the_form.length) {
         }
     }
 
+    $('.breadcrumb-item a').each(function(key, value) {
+        var valObj = $(value)
```

Comment:
here usually we do not define `key`, `value`, and directly use `var $valObj = $(this);`